src/game/default/g_main.c

```diff
@@ -1309,55 +1334,75 @@ void G_Init(void) {
 	g_hook_distance = gi.Cvar("g_hook_distance", va("%.1f", PM_HOOK_DEF_DIST), CVAR_SERVER_INFO,
-							  "The maximum distance the hook will travel");
+							  "The maximum distance the hook will travel.");
+	g_hook_refire = gi.Cvar("g_hook_refire", "0.25", CVAR_SERVER_INFO,
+	                       "The refire delay on the grapple hook in seconds.");
 	g_hook_style = gi.Cvar("g_hook_style", "default", CVAR_SERVER_INFO,
 	                       "Whether to allow only \"pull\", \"swing\" or any (\"default\") hook swing style.");
-	g_hook_speed = gi.Cvar("g_hook_speed", "900", CVAR_SERVER_INFO, "The speed that the hook will fly at");
-	g_hook_pull_speed = gi.Cvar("g_hook_pull_speed", "700", CVAR_SERVER_INFO,
-	                            "The speed that you get pulled towards the hook");
-	g_frag_limit = gi.Cvar("g_frag_limit", "30", CVAR_SERVER_INFO, "The frag limit per level");
-	g_friendly_fire = gi.Cvar("g_friendly_fire", "1", CVAR_SERVER_INFO, "Enables friendly fire");
-	g_force_demo = gi.Cvar("g_force_demo", "0", CVAR_SERVER_INFO, "Force all players to record a demo");
-	g_force_screenshot = gi.Cvar("g_force_screenshot", "0", CVAR_SERVER_INFO, "Force all players to take a screenshot");
-	g_gameplay = gi.Cvar("g_gameplay", "default", CVAR_SERVER_INFO, "Selects deathmatch, duel, arena, or instagib combat");
+	g_hook_speed = gi.Cvar("g_hook_speed", "1200", CVAR_SERVER_INFO, "The speed that the hook will fly at.");
+	g_hook_pull_speed = gi.Cvar("g_hook_pull_speed", "800", CVAR_SERVER_INFO,
+	                            "The speed that you get pulled towards the hook.");
+	g_frag_limit = gi.Cvar("g_frag_limit", "30", CVAR_SERVER_INFO, "The frag limit per level.");
+	g_friendly_fire = gi.Cvar("g_friendly_fire", "1.0", CVAR_SERVER_INFO, "Factor of how much damage can be dealt to teammates.");
+	g_force_demo = gi.Cvar("g_force_demo", "0", CVAR_SERVER_INFO, "Force all players to record a demo.");
+	g_force_screenshot = gi.Cvar("g_force_screenshot", "0", CVAR_SERVER_INFO, "Force all players to take a screenshot.");
+	g_gameplay = gi.Cvar("g_gameplay", "default", CVAR_SERVER_INFO, "Selects deathmatch, duel, arena, or instagib combat.");
 	g_gravity = gi.Cvar("g_gravity", "800", CVAR_SERVER_INFO, NULL);
 	g_handicap = gi.Cvar("g_handicap", "1", CVAR_SERVER_INFO,
 	                     "Allows usage of player handicap. 0 disallows handicap, 1 allows handicap, 2 allows handicap but disables damage reduction. (default 1)");
 	g_inhibit = gi.Cvar("g_inhibit", "", CVAR_SERVER_INFO,
 	                    "Prevents entities from spawning using a class name filter (e.g.: \"weapon_bfg ammo_nukes item_quad\")");
-	g_num_teams = gi.Cvar("g_num_teams", "default", CVAR_SERVER_INFO, "The number of teams allowed. By default, picks the valid amount for the map, or 2.");
-	g_match = gi.Cvar("g_match", "0", CVAR_SERVER_INFO, "Enables match play requiring players to ready");
+	g_num_teams = gi.Cvar("g_num_teams", "default", CVAR_SERVER_INFO,
```

Comment:
I prefer this stays named `g_teams`

Reply:
Well I didn't touch this :thinking:

Reply:
Oh yea! I see. So I think `g_teams` currently controls team deathmatch specifically, and then `g_num_teams` enforces team count for both TDM and CTF.

We *could* use `g_teams` and `g_ctf` to each specify the number of teams:

 * `g_teams 1|2` Would enable normal TDM
 * `g_teams 3|4` Would enable 3-4 team TDM

Same for `g_ctf`. Thoughts? Which is more usable / friendlier?

Reply:
Probably the current method. Also I thought you could have 3/4 player TDM?

Reply:
You can, yes. It's just CTF without the flags essentially.
I don't like the idea because it means that g_teams and g_ctf then become mutually exclusive; one can't be set without the other/the other will reset to 0 if it's set. It's much easier to have CTF be a boolean on/off and keep teams the way it is, that way you can just easily switch CTF off/on at any time/change team count at any time without touching the other cvar.

Reply:
Ah, that's a great point. Let's leave it, then.